mattermost-plugin/webapp/src/index.tsx

```diff
@@ -408,11 +407,3 @@ export default class Plugin {
         this.registry?.unregisterWebSocketEventHandler(wsClient.clientPrefix + ACTION_UPDATE_BLOCK)
     }
 }
-
-declare global {
-    interface Window {
-        registerPlugin(id: string, plugin: Plugin): void
-    }
-}
-
-window.registerPlugin(manifest.id, new Plugin())
```

Comment:
This isn't needed as a product because the web app doesn't use that part of the plugin architecture to load Boards as a product

Reply:
But what happens when we compile it in plugin mode?

Reply:
nevermind, I've seen the other file.